### PR TITLE
TTZHTo4b cards

### PR DESCRIPTION
**Pull request by wweiphy**

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTZHTo4b_5f_LO/TTZHTo4b_5f_LO_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+# Automatic ptj and mjj cuts if xqcut > 0
+# (turn off for VBF and single top processes)
+#**********************************************************
+  .false.	=  auto_ptj_mjj   ! Automatic setting of ptj and mjj
+#**********************************************************
+#                                                                    
+#**********************************
+# BW cutoff (M+/-bwcutoff*Gamma)
+#**********************************
+  15000.0	=  bwcutoff       ! (M+/-bwcutoff*Gamma)
```
Comment (mlizzo) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTZHTo4b_5f_LO/TTZHTo4b_5f_LO_run_card.dat:93]: Please set bwcutoff to 15

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTZHTo4b_5f_LO/TTZHTo4b_5f_LO_customizecards.dat [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#put card customizations here (change top and higgs mass for example)
+set param_card mass 6 172.5
+set param_card yukawa 6 172.5
+set param_card mass 25 125.0
```
Comment (mlizzo) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTZHTo4b_5f_LO/TTZHTo4b_5f_LO_customizecards.dat:4]: Same comment as in PR https://github.com/cms-sw/genproductions/pull/3189 , I don't think this card is actually needed in this case as well

**Comment by mlizzo** [comment]
I don't have further comments, please @menglu21 @Saptaparna @GurpreetSinghChahal have a look at it

**Comment by menglu21** [comment]
Hi @wweiphy , do you plan to decay top and higgs using pythia

**Comment by wweiphy** [comment]
@menglu21 Yes, I was told not to decay higgs at lhe level.

**Comment by menglu21** [comment]
> @menglu21 Yes, I was told not to decay higgs at lhe level.

and also the top? the top usually decayed using madspin

**Comment by wweiphy** [comment]
> and also the top? the top usually decayed using madspin

Ok, I'll add that.

**Comment by menglu21** [comment]
thanks for the update, they look fine to me now. @Saptaparna @GurpreetSinghChahal @mlizzo please have a check and feel free to merge.

**Comment by sihyunjeon** [comment]
@wweiphy  One question - this is going into mg242legacy branch. Why?

@menglu21 @mlizzo

**Comment by wweiphy** [comment]
@sihyunjeon We would like to have the MC in pre-legacy format and we were told by the MC contact that we should push the cards into 242 legacy.

**Comment by wweiphy** [comment]
> @wweiphy what was the reason for this decision? Can we not generate with 2.6.5 (current master) even if the detector conditions are that of EOY?

Just to keep consistency. The other MC we use currently are all pre-legacy.

**Pull request merged**